doc/changelog/05-tactic-language/15741-ltac2-apply.rst

```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  Ltac2 `apply` and `eapply` not unifying with implicit arguments;
+  unification inconsistent with `exact` and `eexact`.
```

Comment:
Nitpick, but I've read the warning message enough to have been traumatized: there shouldn't be a dot finishing this sentence, it's already there two lines below.